src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -1093,7 +1093,7 @@
 
   "com.minecolonies.coremod.general.canttakeequipped": "The only armor you have is what you're wearing. Put it in your inventory if you want to give it to me.",
 
-  "com.minecolonies.coremod.gui.workerhuts.siftersettings": "Daily limit",
+  "com.minecolonies.coremod.gui.workerhuts.siftersettings": "Daily limit progress",
```

Comment:
Bit confusing. What does this actually do? Maybe a screenshot?

Reply:
![image](https://user-images.githubusercontent.com/37391233/110517298-8695b880-80bf-11eb-8bf9-3b88fb689e4f.png)
The daily limit goes up automatically with building level now, and it shows how much of the daily limit has been used.

Reply:
If that means you're not typing in the daily limit any more and it just always targets the max limit, it probably shouldn't have that border.

Reply:
I guess you can still decrease the daily limit yourself, it's just that the max limit is different is what I understood.

Reply:
No, the text box was left a text box, but disabled. I'm not 100% convinced that we won't add back the manual setting of the limit, but I want playtest feedback. uecasm is right, in the long run. The value in the text box is how many blocks sifted so far today.